lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +218,23 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        return [ConlluSentence(idx, sentence, []) for idx, sentence in enumerate(split_by_sentence(text))]
```

Comment:
why do you leave tokens as empty list?

Reply:
i couldn't understand the logic behind this method but now i do, fixed